src/rivian/rivian.py

```diff
@@ -278,6 +281,12 @@ async def enroll_phone(
             if data.get("data", {}).get("enrollPhone", {}).get("success"):
                 return True
         return False
+    
+    async def ble_pair_phone(
+            self, device: BLEDevice,  vehicle_id: str, phone_id: str, vehicle_key: str, private_key: str
+    ) -> bool:
+        """Connect to and pair a phone key via BLE."""
+        return await pair_phone(device, vehicle_id, phone_id, vehicle_key, private_key)
```

Comment:
I'm wondering if we should actually remove the ble references entirely from this file and just let the consuming client import from `rivian.ble` should they choose to implement it.